README.md

```diff
@@ -39,7 +39,7 @@ response = requests.post(url, data=payload, headers=client_headers)
 ```
 
 - _url_ is the service endpoint (for example:
-  `https://dat-a.osisoft.com`). The connection is used by the `SdsClient` class.
+  `https://uswe.datahub.connect.aveva.com`). The connection is used by the `SdsClient` class.
```

Comment:
Updated these since they are shown in the text

Reply:
also need to update the actual appsetting.placeholder.json

Reply:
We can add a secret to the Azure key vault with the old resource and modify the pipeline yaml to include it as one of the retrieved secrets.

Reply:
Okay, I misunderstood it as we aren't updating the resource until that environment is set up and ready, but Collin's suggestion is good since that wouldn't break all the pipelines.